drivers/bridge/bridge.go

```diff
@@ -432,6 +460,9 @@ func (d *driver) CreateNetwork(id types.UUID, option map[string]interface{}) err
 			d.Unlock()
 			return types.ForbiddenErrorf("conflicts with network %s (%s)", nw.id, nw.config.BridgeName)
 		}
+		if nw.isDefaultNetwork() {
+			nw.config.copyDefaultConfigs(option, config)
```

Comment:
Conisdering the above comment, can we change this to

config.copyFromDefaultConfig(nw)